app/src/main/java/org/mifos/mobilebanking/utils/DateHelper.java

```diff
@@ -46,12 +46,13 @@
     public static String getDateAsString(List<Integer> integersOfDate) {
 
         StringBuilder stringBuilder = new StringBuilder();
-        stringBuilder.append(integersOfDate.get(2))
-                .append(' ')
-                .append(getMonthName(integersOfDate.get(1)))
-                .append(' ')
-                .append(integersOfDate.get(0));
-
+        if (integersOfDate != null) {
```

Comment:
@MigDinny Why are you doing a null check here?

Reply:
I am not. 

https://github.com/openMF/mifos-mobile/commit/e6ef485669254bb90d1bb14b7f206ba3cdc55bfd

Reply:
@MigDinny Your are not pulling the changes properly. `Green Colors` shows the additions by your branch/PR. Red shows the line you have deleted. Analyze your code properly.